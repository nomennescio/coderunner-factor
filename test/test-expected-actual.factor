! Copyright 2019 nomennescio

USING: tools.testest ;
IN: tests

: run-tests ( -- )

"Expected results and actual results tests" describe#{
  "Match" it#{
     <{ 0 1 2 3 -> 0 1 2 3 }>
  }#
  "Mismatch" it#{
     <{ 0 1 2 3 -> 4 5 6 7 }>
  }#
}#
;

MAIN: run-tests

