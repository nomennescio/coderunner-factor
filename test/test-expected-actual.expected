
<DESCRIBE::>Expected results and actual results tests

<IT::>Match

<PASSED::>Test Passed

<COMPLETEDIN::>0.025956 ms

<IT::>Mismatch

<FAILED::>Test Failed : Expected :<:LF:>4<:LF:>5<:LF:>6<:LF:>7<:LF:>but got :<:LF:>0<:LF:>1<:LF:>2<:LF:>3

<COMPLETEDIN::>0.126585 ms

<COMPLETEDIN::>0.382813 ms
